src/node_quic_util.h

```diff
@@ -200,7 +200,7 @@ class SocketAddress {
     }
 
     char host[NI_MAXHOST];
-    if (uv_inet_ntop(af, binaddr, host, sizeof(host)) == 0)
+    if (uv_inet_ntop(af, binaddr, host, sizeof(host)) != 0)
```

Comment:
This seems to be missed from the last rebasing. Refs: https://github.com/nodejs/quic/pull/139/files

Reply:
how odd... thanks for catching